doc/releases/release-notes-3.6.rst

```diff
@@ -29,6 +29,10 @@ Architectures
 
 * ARC
 
+ * Extended support for ARCv3 features: enable HW prefetch and shared cache;
+ * Disabled TLS support for platforms with two or more register banks;
```

Comment:
Please explain what that TLS is. I guess here it stands for "Thread-local Storage". But there're other uses of the same acronym like "Transport Layer Security" etc. Let's not confuse a reader here.